clients/android/app/src/main/java/app/lockbook/utils/Structures.kt

```diff
@@ -51,21 +51,27 @@ data class DecryptedValue(
 
 data class Account(
     val username: String,
-    val keys: String
+    val keys: RSAPrivateKey
+)
+
+data class RSAPrivateKey(
```

Comment:
I am concerned about overflow rsa people are using this library to represent these values: https://docs.rs/num-bigint/0.3.0/num_bigint/

I'm wondering if you even need to serialize this value or if you could just leave it as a json string or something

Reply:
I don't know if I could leave just that particular part in a json string, but this should work. This the recommended format that Klaxon wanted me to use, and I am unsure why you believe it can't represent these large values. It just splits the values, doesn't it? I assume it still works since I can execute work successfully